.github/container/test-maxtext.sh

```diff
@@ -163,6 +164,10 @@ if [ $MODEL != "gpt3-52k" ]; then # gpt3-52k only works with dot_product
     ADDITIONAL_ARGS+=" attention=${ATTN_TYPE}"
 fi
 
+if [ ${ATTN_TYPE} == "cudnn_flash_te" ]; then
```

Comment:
Assert that it is a value that is supported?

Reply:
No, TE requires us to set the `NVTE_FUSED_ATTN=1` in order to use the fused attention api. This check is to set the environment variable. For other attn types, we don't need this